src/main/scala/firrtl/transforms/formal/RemoveVerificationStatements.scala

```diff
@@ -0,0 +1,41 @@
+
+package firrtl.transforms.formal
+
+import firrtl.ir.{Circuit, EmptyStmt, Statement, Verification}
+import firrtl.{CircuitState, DependencyAPIMigration, MinimumVerilogEmitter, Transform, VerilogEmitter}
+import firrtl.options.{Dependency, PreservesAll}
+import firrtl.stage.TransformManager.TransformDependency
+
+
+/**
+  * Remove Verification Statements
+  *
+  * Replaces all verification statements in all modules with the empty statement.
+  * This is intended to be required by the Verilog emitter to ensure compatibility
+  * with the Verilog 2001 standard.
+  */
+class RemoveVerificationStatements extends Transform
+  with DependencyAPIMigration
+  with PreservesAll[Transform] {
+
+  override def prerequisites: Seq[TransformDependency] = Seq.empty
+  override def optionalPrerequisites: Seq[TransformDependency] = Seq.empty
+  override def optionalPrerequisiteOf: Seq[TransformDependency] =
+    Seq( Dependency[VerilogEmitter],
+      Dependency[MinimumVerilogEmitter])
+
+  def removeVerification(s: Statement): Statement = s match {
+    case _: Verification => EmptyStmt
```

Comment:
This should emit a warning (does `logger.warn(....)` work?) the first time a statement is replaced.
The warning should point the user to the SystemVerilog emitter and explain how to enable it.